src/frontend/src/lib/utils/format.utils.ts

```diff
@@ -68,6 +68,30 @@ export const formatNanosecondsToDate = (nanoseconds: bigint): string => {
 	return date.toLocaleDateString('en', DATE_TIME_FORMAT_OPTIONS);
 };
 
+export const formatSecondsToNormalizedDate = ({
+	seconds,
+	currentDate
+}: {
+	seconds: number;
+	currentDate?: Date;
+}): string => {
+	const date = new Date(seconds * 1000);
+	const today = currentDate ?? new Date();
```

Comment:
Naming question: when is current date not the current date or when is current date provided not as the current date?

Reply:
the idea is to provide the date of reference, so if used in a loop, all transactions have the same base to calculate. If you want, i can remove the `const today = currentDate ?? new Date();` and just make the user provide the today date as input always

Reply:
No no, now I understand. Maybe a comment to describe this usage?

Reply:
Sorry, I missed in this PR; done with:

- #3543